app/middlewares/validate-dto.js

```diff
@@ -0,0 +1,13 @@
+const { validationError } = require('../errors');
+
+function validateDto(ajvValidate) {
```

Comment:
Why is this defined as a classic function? Why not a function in a variable? Why not an anonymous function (arrow function)?

Reply:
I am not saying it is bad, just want to talk about the differences.

Reply:
mmm, I didn't think about it, I know that the difference between classic functions and arrow functions is the scope of "this" and that arrow functions have an implicit return. I am not sure but, I think that for this case it doesn't affect. I rewrote the function in this way using arrow functions, do you think it is better?    

```
  const validateDto = ajvValidate => (req, res, next) => {
  const valid = ajvValidate(req.body);
  if (!valid) {
    const { errors } = ajvValidate;
    return next(validationError(errors));
  }
  return next();
};
```

Reply:
I think it is the same for this case, but we usually use const vars for functions declarations, even for anon or classic type. It was just a question to see if you know that 😛